tests/lib/rules/no-wait-for-side-effects.test.ts

```diff
@@ -722,6 +735,41 @@ ruleTester.run(RULE_NAME, rule, {
       `,
 				errors: [{ line: 4, column: 11, messageId: 'noSideEffectsWaitFor' }],
 			} as const,
+			{
+				// Issue #500, https://github.com/testing-library/eslint-plugin-testing-library/issues/500
+				code: `
+        import { waitFor } from '${testingFramework}';
+        await waitFor(function() {
```

Comment:
Could you remove the `await` operator from the `waitFor` utils chained with `then`? So we can make sure the example is valid because thenable promises are handled properly, and not because it was a side-effect from the `wait` operator.

Reply:
Good catch! Done.

Reply:
Removed the `await` also from the rule documentation example.